package.json

```diff
@@ -7,7 +7,7 @@
     "update-node": "./bin/update-node.js"
   },
   "engines": {
-    "node": "^18.17.1"
+    "node": ">=18.12.1"
```

Comment:
any reason why here it starts with the minor 12? I saw 18.14.2 below on other places

Reply:
Almost kind of random. 
- 18.12.1 is the oldest 18 LTS
- 18.14.2 was the oldest nvm installed version for node 18 on my device